src/type-translator.ts

```diff
@@ -270,6 +270,9 @@ export class TypeTranslator {
         typeStr += this.translate(referenceType.target);
       }
       if (referenceType.typeArguments) {
+        // Closure doesn't accept a type like {?<...>}, so if the parameterized
```

Comment:
This code is generating the output for parameterized types e.g. `FooType<string, number>`.  If the `FooType` in particular is `?`, we would generate something like `?<string, number>` which is illegal in Closure.

I think this is a broader tsickle bug in typed mode that this change reveals because it exposes more tsickle logic in untyped mode.

Do you have any better suggestion on how to word this?  I agree it kinda sucks right now.